server/database/config/fakeData.sql

```diff
@@ -0,0 +1,60 @@
+BEGIN;
+
+DROP TABLE IF EXISTS admin,cohort,student,project,studentProject CASCADE;
+
+CREATE TABLE admin 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    username VARCHAR(100) NOT NULL ,
+    password VARCHAR(100) NOT NULL 
+);
+
+CREATE TABLE cohort 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    name VARCHAR(100) NOT NULL ,
+    description TEXT NOT NULL ,
+    imgUrl TEXT NOT NULL ,
+    githubLink TEXT NOT NULL 
+);
+
+CREATE TABLE student 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
+    name VARCHAR(100) NOT NULL ,
```

Comment:
please add a column for student email after name column:
```suggestion
    name VARCHAR(100) NOT NULL,
    email VARCHAR(200) NOT NULL UNIQUE,
```